scripts/oracle/getPrices_chainlink.js

```diff
@@ -1,47 +0,0 @@
-// Smart contracts
```

Comment:
How can we see the current price for a given oracle (from chainlink)?

Reply:
This is all done in the `scripts/oracle/getPrices.js` file.